src/components/Header/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import NavDropdown from '../../containers/NavDropdown';
+import NavProfile from '../../containers/NavProfile';
+
+import logo from '../../assets/graphics/logo.svg';
+
+import './style.less';
+
+const Header: React.FC = () => {
+  return (
+    <div className="header-container">
+      <div className="header-content">
+        <div className="header">
+          <NavLink className="title" to="/">
+            <img className="logo" src={logo} alt="ACM Logo" />
+            <span className="subheading">&nbsp;Membership Portal</span>
```

Comment:
The top bar text and acm logo are too close together, maybe add `margin: 0 1em` to the subheading class style

![Uploading image.png…]()